src/constants.c

```diff
@@ -0,0 +1,40 @@
+#include <sys/epoll.h>
+#include <sys/eventfd.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+void constant(const char* name, int value) {
+  printf("  let %s = 0x%x\n", name, value);
+}
+
+int main(int argc, char **argv) {
+  printf("module Epoll_constants = struct\n");
```

Comment:
The target module is `constants.ml`; thus the internal modules could be simply `Epoll` and `EventFD` (matching the names on the ML side`

Reply:
done